### PR TITLE
US-1073: Add OrderStatus page and Routing

### PR DESCRIPTION
**Pull request by JeremySparkman**
Added dummy data variable to Route and I also fixed a failing Routes test.

### CODE REVIEW

File: app/components/OrderStatus/index.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default (): React.ReactElement => {
+  return <div>Order Status</div>;
+};
```
Comment (jmiller-nexient) [inline app/components/OrderStatus/index.tsx:5]: Be consistent with the structure we've used for the other components and name it here as a const and have the `export` statement at the end of the file.

Comment (jmiller-nexient) [reply app/components/OrderStatus/index.tsx:5]: I think we should also include "Page" at the end of the name so it is clear this component is intended to be a page and not reused within other pages.

File: app/components/Routes/index.tsx [hunk 2]
```diff
@@ -13,6 +17,9 @@ const Routes: React.FC<{}> = (): React.ReactElement => (
     <Route path="/testpage">
       <TestPage />
     </Route>
+    <Route path={`/orders/${orderNumber}`}>
+      <OrderStatus />
+    </Route>
```
Comment (jmiller-nexient) [inline app/components/Routes/index.tsx:22]: Put this in alphabetic order by the path ('/o' comes before '/t')

File: app/components/Routes/tests/index.test.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { mount } from 'enzyme';
 
 import { translationMessages } from 'i18n';
 
-import Home from 'components/Layout';
+import Home from 'components/Home';
```
Comment (jmiller-nexient) [inline app/components/Routes/tests/index.test.tsx:8]: That would 'splain it!

**Comment by JeremySparkman** [comment]
Updated based on feedback

**Pull request merged**